openpype/hosts/traypublisher/plugins/publish/collect_explicit_colorspace.py

```diff
@@ -13,9 +14,14 @@ class CollectColorspace(pyblish.api.InstancePlugin,
     order = pyblish.api.CollectorOrder + 0.49
     hosts = ["traypublisher"]
 
+    colorspace_items = [
+        (None, "Don't override")
+    ]
+
     def process(self, instance):
         values = self.get_attr_values_from_data(instance.data)
         colorspace = values.get("colorspace", None)
+        self.log.debug("colorspace: {}".format(colorspace))
         if not colorspace:
             return
```

Comment:
Should we log this only after `if not colorspace: return` below so we don't log "Don't override"?


```suggestion
        if colorspace is None:
            return

        self.log.debug("Explicit colorspace set to: {}".format(colorspace))
```